src/twisted/web/http_headers.py

```diff
@@ -97,13 +98,13 @@ def __cmp__(self, other):
             )
         return NotImplemented
 
-    def _encodeName(self, name):
+    def _encodeName(self, name: String) -> bytes:
```

Comment:
yeah AnyStr here would definitely work better

Reply:
I went with  `Union[bytes, str]` here.  Do you want that, or `AnyStr`?

Reply:
I went with AnyStr as suggested by @wsanchez

Reply:
In this case, it doesn't need to be `AnyStr`, since the return value and `name` don't have to match.  That said, I think it's probably OK to use here… you just need to be careful that `AnyStr` != `Union[bytes, str] ` in that the types must be the same.